### PR TITLE
Base service

### PR DESCRIPTION
**Pull request by Victor-C-Zhang**
Above. Create a new wrapper service to encapsulate
- wifi
- event loop
- config reader
- graphics
- TODO: BT connectivity + update requests from companion app

### CODE REVIEW

File: device/linux/wifi/wifi.cpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ wifi_err_t wifi_disconnect() {
   return RES_OK;
 }
 
-bool wifi_is_disconnected() {
+bool wifi_is_connected() {
```
Comment (q349wang) [inline device/linux/wifi/wifi.cpp:35]: Default true?

File: base/src/service.cpp [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+#include "antioch/base/config/configerator.h"
+
+namespace antioch::base {
+
+using antioch::connectivity::wifi::WifiService;
+using antioch::connectivity::wifi::WifiConnectionError;
+
+void Service::start() {
+  if (!gfx_init()) {
```
Comment (q349wang) [inline base/src/service.cpp:16]: Kek

**Review by q349wang — APPROVED** [review]
👍👍👍👍👍

File: device/linux/wifi/wifi.cpp (hunk 1 above)
Comment (Victor-C-Zhang) [reply device/linux/wifi/wifi.cpp:35]: this is a stub impl for my laptop. so yes wifi is connected

**Pull request merged**